client/top-100/top-100Script.js

```diff
@@ -53,6 +53,34 @@ document.addEventListener('DOMContentLoaded', () => {
     }));
     return top100moviesArray;
   }
-
-  imageExtractor(top100List);
-});
\ No newline at end of file
+  const movieList = ['Avengers:+Endgame', 'The+Dark+Knight', 'Django+Unchained', 'Spider-Man:+Into+the+Spider-Verse']
+  const imageList = [];
+  async function getImage() {
+    for (let i = 0; i < movieList.length; i++) {
+      const frontUrl = 'https://api.themoviedb.org/3/search/movie?api_key=ee671a8f0facbafb2b7da0decd7157db&query='
+      const results = await fetch(frontUrl+`${movieList[i]}`);
+      const movieResults = await results.json();
+      console.log(movieResults.results[0].backdrop_path);
+      const backdropPath = movieResults.results[0].backdrop_path;
+      const imageRequest = await fetch(`https://image.tmdb.org/t/p/w342/${backdropPath}`);
+      const img = await imageRequest.blob();
+      imageList.push(img);
+      console.log(imageRequest);
+    }
+  }
+  getImage()
```

Comment:
const imageSlides = getImage(imageList);

Reply:
This explains why it was null because it wasn't  returning any arraylist. Make sense? @ngaffney22